source/material_model/visco_plastic.cc

```diff
@@ -1072,38 +1072,31 @@ namespace aspect
                                    "$\\hat{D} = \\hat{D_{e}} + \\hat{D_{v}}$.  "
                                    "These terms further decompose into "
                                    "$\\hat{D_{v}} = \\frac{\\tau}{2\\eta}$ and "
-                                   "$\\hat{D_{e}} = \\frac{\\overset{\\triangledown}{\\tau}}{2\\mu}$, where "
+                                   "$\\hat{D_{e}} = \\frac{\\overset{\\nabla}{\\tau}}{2\\mu}$, where "
```

Comment:
I think this and the other changes in this file are from a different branch, which I believe we already merged. Can you just rebase to current master?

Reply:
I have never rebased before.  I did this:
 % git rebase master
Current branch clarify_refinement is up to date.

Reply:
Note that this text appears in two files visco-elastic.cc and viscoplastic.cc. Maybe you are the changes were done in one file previously but the other. The point here was to make them exactly the same where they are describing the same set of
 equations

Reply:
Try this:
```
git checkout master
git pull upstream master
git checkout clarify_refinement
git rebase master
git push -f origin clarify_refinement
```
and see what happens.

Reply:
I have done this now. That was cool!

Reply:
I was wrong: I had thought that #3670 had already been merged. You have the same changes there that you have here. Let's wait until #3670 is merged, and then you'll have to do the same steps as above again.

Reply:
Okay - I see what I did wrong now. I should not have included this file with this pull request. I'm not exactly sure how that happened, but I walk through the steps more slowly for the next pull request. Thank you.